Remotion/Data/DomainObjects/Persistence/Rdbms/SqlServer/Sql2016/SqlStorageObjectFactory.cs

```diff
@@ -161,7 +161,14 @@ public virtual IStorageTypeInformationProvider CreateStorageTypeInformationProvi
     {
       ArgumentUtility.CheckNotNull("rdmsStorageProviderDefinition", rdmsStorageProviderDefinition);
 
-      return new SqlStorageTypeInformationProvider();
+      return new SqlStorageTypeInformationProvider(CreateDateTimeDefaultStorageTypeProvider(rdmsStorageProviderDefinition));
```

Comment:
readablity / consistency: I think we use local variables for the services before injecting them. It becomes relevant if there's more services and makes it more explicit / obvious in general. If it's not done now, it's gonna be done the next time we touch this ctor.